--- fig2dev-3.2.6a/fig2dev/dev/genemf.c.orig	2017-01-07 23:07:15.000000000 +0100
+++ fig2dev-3.2.6a/fig2dev/dev/genemf.c	2017-07-29 18:19:49.027329674 +0200
@@ -696,7 +696,7 @@
 static void shape();
 static void shape_interior();
 static void textunicode();
-static void text();
+static void textfunc();
 static void textcolr();
 static void textalign();
 #if defined(I18N) && defined(HAVE_ICONV)
@@ -3020,7 +3020,7 @@
 
 
 static void
-text(
+textfunc(
 	int x,
 	int y,		/* reference point */
 	RECTL *bbx,	/* bounding box */
@@ -3506,7 +3506,7 @@
 		bak = *s1;
 		*s1 = '\0';
 
-		text(0, 0, &rclBounds, s);
+		textfunc(0, 0, &rclBounds, s);
 
 		*s1 = bak;
 	    }
@@ -3532,7 +3532,7 @@
     }
 
     textfont(font, t->size, round(t->angle * RAD_01DEG));
-    text(t->base_x, t->base_y, &rclBounds, t->cstring);
+    textfunc(t->base_x, t->base_y, &rclBounds, t->cstring);
 }/* end genemf_text */
 
 
