diff -Nru transfig.3.2.3d/fig2dev/Imakefile transfig.3.2.3d.new/fig2dev/Imakefile
--- transfig.3.2.3d/fig2dev/Imakefile	Thu Apr 19 19:19:38 2001
+++ transfig.3.2.3d.new/fig2dev/Imakefile	Sat Jun 16 21:31:12 2001
@@ -28,7 +28,7 @@
 XCOMM You should point XFIGLIBDIR to the same directory you did when you compiled
 XCOMM and installed xfig.  This will be made automatic in the future.
 
-XFIGLIBDIR =	/usr/local/lib/X11/xfig
+XFIGLIBDIR =	/usr/X11R6/share/xfig
 
 XCOMM ******
 XCOMM If your system has the strerror() function (doesn't have sys_errlist) then
@@ -56,7 +56,7 @@
 XCOMM the full name (rgb.txt) now.
 XCOMM
 
-RGB = $(LIBDIR)/rgb.txt
+RGB = /usr/X111R6/lib/X11/rgb.txt
 
 XCOMM Comment out the following #define USEPNG if you don't want PNG support
 XCOMM Uncomment the #define for USEPNG if you want to be able to import
@@ -82,7 +82,7 @@
 
 #ifdef USEXPM
 XPMLIBDIR = $(USRLIBDIR)
-XPMINC = -I/usr/include/X11
+XPMINC = -I/usr/X11R6/include/X11
 #endif
 
 XCOMM ****************
@@ -104,8 +104,8 @@
 
 #ifdef USEJPEG
 #ifdef USEINSTALLEDJPEG
-JPEGLIBDIR = /usr/local/lib
-JPEGINCDIR = /usr/include/X11
+JPEGLIBDIR = /usr/lib
+JPEGINCDIR = /usr/include
 #else
 JPEGLIBDIR = ../jpeg
 JPEGINCDIR = $(JPEGLIBDIR)
@@ -158,11 +158,11 @@
 
 #ifdef I18N
 I18N_DEFS = -DI18N 
-FIG2DEV_LIBDIR = /usr/local/lib/fig2dev
+FIG2DEV_LIBDIR = /usr/X11R6/share/fig2dev
 I18N_DEV_DEFS = $(I18N_DEFS) -DFIG2DEV_LIBDIR=\\\"$(FIG2DEV_LIBDIR)\\\"
 
 install::
-	[ -d $(DESTDIR)$(FIG2DEV_LIBDIR) ] || mkdir $(DESTDIR)$(FIG2DEV_LIBDIR)
+	[ -d $(DESTDIR)$(FIG2DEV_LIBDIR) ] || mkdir -p $(DESTDIR)$(FIG2DEV_LIBDIR)
 	$(INSTALL) -m 644 -c dev/japanese.ps $(DESTDIR)$(FIG2DEV_LIBDIR)
 	( cd $(DESTDIR)$(FIG2DEV_LIBDIR); \
 	  rm -f ja.ps ja_JP.ps ja_JP.ujis.ps ja_JP.eucJP.ps ja_JP.EUC.ps; \
@@ -185,7 +185,7 @@
 XCOMM *** You shouldn't have to change anything below this point ***
 XCOMM **************************************************************
 
-DIR_DEFS=	-DBITMAPDIR=\\\"$(DESTDIR)$(XFIGLIBDIR)/bitmaps\\\"
+DIR_DEFS=	-DBITMAPDIR=\\\"$(DESTDIR)/usr/X11R6/share/xfig\\\"
 
 #ifdef USEPNG
 DUSEPNG = -DUSE_PNG
@@ -280,11 +280,11 @@
 XCOMM Install the tk bitmaps here
 install::
 	@(cd bitmaps ; \
-	if [ -d $(DESTDIR)$(XFIGLIBDIR)/bitmaps ]; then set +x; \
-	else (set -x; $(MKDIRHIER) $(DESTDIR)$(XFIGLIBDIR)/bitmaps ); fi ; \
-	echo Copying bitmap files for tk to $(DESTDIR)$(XFIGLIBDIR)/bitmaps ; \
+	if [ -d $(DESTDIR)/usr/X11R6/share/xfig ]; then set +x; \
+	else (set -x; $(MKDIRHIER) $(DESTDIR)/usr/X11R6/share/xfig ); fi ; \
+	echo Copying bitmap files for tk to $(DESTDIR)/usr/X11R6/share/xfig ; \
 	for f in * ; do \
-	      $(INSTALL) -m 644 -c $$f $(DESTDIR)$(XFIGLIBDIR)/bitmaps ; \
+	      $(INSTALL) -m 644 -c $$f $(DESTDIR)/usr/X11R6/share/xfig ; \
 	done) ; 
 
 InstallScript(fig2ps2tex,$(BINDIR))
