diff -Nur --exclude *~ transfig.3.2.3c/fig2dev/dev/genepic.c transfig.3.2.3c.new/fig2dev/dev/genepic.c
--- transfig.3.2.3c/fig2dev/dev/genepic.c	Tue Jul 11 23:50:28 2000
+++ transfig.3.2.3c.new/fig2dev/dev/genepic.c	Tue Feb  6 17:07:20 2001
@@ -1302,9 +1302,6 @@
 	      }
 	    }
 	    else
-	        if (*cp >= 0xa0)	/* we escape 8-bit char */
-	    		fprintf(tfp, "%s", ISOtoTeX[(int)*cp-0xa0]);
-		else
 			fputc(*cp, tfp);
 	  }
     fprintf(tfp, "}}}}}\n");
diff -Nur --exclude *~ transfig.3.2.3c/fig2dev/dev/genlatex.c transfig.3.2.3c.new/fig2dev/dev/genlatex.c
--- transfig.3.2.3c/fig2dev/dev/genlatex.c	Thu Jul 20 01:16:38 2000
+++ transfig.3.2.3c.new/fig2dev/dev/genlatex.c	Tue Feb  6 17:07:08 2001
@@ -800,9 +800,6 @@
 			fputc(*cp, tfp);
 		    else
 #endif
-		    if (*cp >= 0xa0)
-			 fprintf(tfp, "%s", ISOtoTeX[(int)*cp-0xa0]);
-		else
 		    fputc(*cp, tfp);
 		}
 
diff -Nur --exclude *~ transfig.3.2.3c/fig2dev/dev/genpictex.c transfig.3.2.3c.new/fig2dev/dev/genpictex.c
--- transfig.3.2.3c/fig2dev/dev/genpictex.c	Wed May  3 22:47:04 2000
+++ transfig.3.2.3c.new/fig2dev/dev/genpictex.c	Tue Feb  6 17:07:33 2001
@@ -639,9 +639,6 @@
 	      	}
 	else 
 		for(cp = (unsigned char*)t->cstring; *cp; cp++) {
-		    if (*cp >= 0xa0)
-		         fprintf(tfp, "%s", ISOtoTeX[(int)*cp-0xa0]);
-		else
 		    fputc(*cp, tfp);
 		}
 
