diff -dur transfig.3.2.5e.orig/fig2dev/dev/gencgm.c transfig.3.2.5e/fig2dev/dev/gencgm.c
--- transfig.3.2.5e.orig/fig2dev/dev/gencgm.c	2003-09-20 01:03:48.000000000 +0200
+++ transfig.3.2.5e/fig2dev/dev/gencgm.c	2013-10-05 19:08:48.115907807 +0200
@@ -139,7 +139,7 @@
   
   if (from) {
 	figname = malloc(strlen(from)+1);
-	sprintf(figname, from);
+	sprintf(figname, "%s", from);
 	p = strrchr(figname, '/');
 	if (p) 
 	    figname = p+1;		/* remove path from name for comment in file */
diff -dur transfig.3.2.5e.orig/fig2dev/dev/genmp.c transfig.3.2.5e/fig2dev/dev/genmp.c
--- transfig.3.2.5e.orig/fig2dev/dev/genmp.c	2007-04-02 23:45:34.000000000 +0200
+++ transfig.3.2.5e/fig2dev/dev/genmp.c	2013-10-05 19:09:24.866177569 +0200
@@ -1150,7 +1150,7 @@
 	    }
 	} else {
 	    /* special text in latex mode: just write the text. */
-	    fprintf(tfp, t->cstring);
+	    fprintf(tfp, "%s", t->cstring);
 	}
 	fprintf(tfp," etex;\n");
 
@@ -1164,7 +1164,7 @@
 
 	fprintf(tfp,"  picture q;\n");
 	fprintf(tfp,"  q=thelabel.urt(\"");
-        fprintf(tfp, t->cstring);
+        fprintf(tfp, "%s", t->cstring);
 	fprintf(tfp, "\" infont ");
 	if (t->font<0) {
 	    fprintf(tfp, "defaultfont");
diff -dur transfig.3.2.5e.orig/fig2dev/trans_spline.c transfig.3.2.5e/fig2dev/trans_spline.c
--- transfig.3.2.5e.orig/fig2dev/trans_spline.c	2004-03-29 20:54:40.000000000 +0200
+++ transfig.3.2.5e/fig2dev/trans_spline.c	2013-10-05 19:08:48.115907807 +0200
@@ -648,7 +648,7 @@
     F_control	   *cp;
 
     if ((cp = (F_control *) malloc(CONTROL_SIZE)) == NULL)
-	fprintf(stderr,Err_mem);
+	fprintf(stderr,"%s",Err_mem);
     return cp;
 }
 
@@ -659,7 +659,7 @@
     F_line	   *l;
 
     if ((l = (F_line *) malloc(LINOBJ_SIZE)) == NULL)
-	fprintf(stderr,Err_mem);
+	fprintf(stderr,"%s",Err_mem);
     l->pic = NULL;
     l->next = NULL;
     l->for_arrow = NULL;
