--- transfig.3.2.4/fig2dev/fig2dev.h.varargs	2003-05-22 23:56:28.000000000 -0400
+++ transfig.3.2.4/fig2dev/fig2dev.h	2003-05-22 23:56:42.000000000 -0400
@@ -22,7 +22,7 @@
 #include <sys/file.h>
 #include <signal.h>
 #include <string.h>
-#include <varargs.h>
+#include <stdarg.h>
 #include <pwd.h>
 #include <errno.h>
 #include <time.h>
--- transfig.3.2.4/fig2dev/dev/gensvg.c.gcc33	2003-05-22 23:58:14.000000000 -0400
+++ transfig.3.2.4/fig2dev/dev/gensvg.c	2003-05-22 23:58:37.000000000 -0400
@@ -692,7 +692,7 @@
     if (t->angle != 0) {
 	fprintf (tfp, "<g transform=\"translate(%d,%d) rotate(%d)\" >\n",
 		 (int) (t->base_x * mag), (int) (t->base_y * mag), degrees (t->angle));
-	fprintf (tfp, "<text x=\"0\" y=\"0\" fill=\"#%6.6x\"  font-family=\"%s\" 
+	fprintf (tfp, "<text x=\"0\" y=\"0\" fill=\"#%6.6x\"  font-family=\"%s\" \n\
 		 font-style=\"%s\" font-weight=\"%s\" font-size=\"%d\" text-anchor=\"%s\" >\n",
 		 rgbColorVal (t->color), family[(int) ceil ((t->font + 1) / 4)],
 		 (t->font % 2 == 0 ? "normal" : "italic"),
@@ -700,7 +700,7 @@
 		 anchor[t->type]);
     }
     else
-	fprintf (tfp, "<text x=\"%d\" y=\"%d\" fill=\"#%6.6x\"  font-family=\"%s\" 
+	fprintf (tfp, "<text x=\"%d\" y=\"%d\" fill=\"#%6.6x\"  font-family=\"%s\" \n\
 		 font-style=\"%s\" font-weight=\"%s\" font-size=\"%d\" text-anchor=\"%s\" >\n",
 		 (int) (t->base_x * mag), (int) (t->base_y * mag), rgbColorVal (t->color),
 		 family[(int) ceil ((t->font + 1) / 4)],
