--- transfig.3.2.4/fig2dev/Imakefile.orig	Thu Dec 19 18:33:44 2002
+++ transfig.3.2.4/fig2dev/Imakefile	Tue Mar 25 01:38:21 2003
@@ -20,7 +20,7 @@
 XCOMM You should point XFIGLIBDIR to the same directory you did when you compiled
 XCOMM and installed xfig.
 
-XFIGLIBDIR =	/usr/local/lib/X11/xfig
+XFIGLIBDIR =	/usr/share/xfig
 
 XCOMM ******
 XCOMM If your system has the strerror() function (doesn't have sys_errlist) then
@@ -48,7 +48,7 @@
 XCOMM the full name (rgb.txt) now.
 XCOMM
 
-RGB = $(LIBDIR)/rgb.txt
+RGB = /usr/share/X11/rgb.txt
 
 XCOMM Comment out the following #define USEPNG if you don't want PNG support
 XCOMM Uncomment the #define for USEPNG if you want to be able to import
@@ -61,7 +61,7 @@
 
 #ifdef USEPNG
 PNGLIBDIR = $(USRLIBDIR)
-PNGINC = -I/usr/include/X11
+PNGINC =
 #endif
 
 XCOMM ****************
@@ -126,11 +126,11 @@
 
 #ifdef I18N
 I18N_DEFS = -DI18N 
-FIG2DEV_LIBDIR = /usr/local/lib/fig2dev
+FIG2DEV_LIBDIR = /usr/share/fig2dev
 I18N_DEV_DEFS = $(I18N_DEFS) -DFIG2DEV_LIBDIR=\\\"$(FIG2DEV_LIBDIR)\\\"
 
 install::
-	[ -d $(DESTDIR)$(FIG2DEV_LIBDIR) ] || mkdir $(DESTDIR)$(FIG2DEV_LIBDIR)
+	[ -d $(DESTDIR)$(FIG2DEV_LIBDIR) ] || mkdir -p $(DESTDIR)$(FIG2DEV_LIBDIR)
 	$(INSTALL) -c dev/japanese.ps $(DESTDIR)$(FIG2DEV_LIBDIR)
 	( cd $(DESTDIR)$(FIG2DEV_LIBDIR); \
 	  rm -f ja.ps ja_JP.ps ja_JP.ujis.ps ja_JP.eucJP.ps ja_JP.EUC.ps; \
@@ -154,7 +154,7 @@
 XCOMM *** You shouldn't have to change anything below this point ***
 XCOMM **************************************************************
 
-DIR_DEFS=	-DBITMAPDIR=\\\"$(DESTDIR)$(XFIGLIBDIR)/bitmaps\\\"
+DIR_DEFS=	-DBITMAPDIR=\\\"$(DESTDIR)$(XFIGLIBDIR)\\\"
 
 #ifdef USEPNG
 DUSEPNG = -DUSE_PNG
@@ -229,11 +229,11 @@
 XCOMM Install the tk bitmaps here
 install::
 	@(cd bitmaps ; \
-	if [ -d $(DESTDIR)$(XFIGLIBDIR)/bitmaps ]; then set +x; \
-	else (set -x; $(MKDIRHIER) $(DESTDIR)$(XFIGLIBDIR)/bitmaps ); fi ; \
-	echo Copying bitmap files for tk to $(DESTDIR)$(XFIGLIBDIR)/bitmaps ; \
+	if [ -d $(DESTDIR)$(XFIGLIBDIR) ]; then set +x; \
+	else (set -x; $(MKDIRHIER) $(DESTDIR)$(XFIGLIBDIR) ); fi ; \
+	echo Copying bitmap files for tk to $(DESTDIR)$(XFIGLIBDIR) ; \
 	for f in * ; do \
-	      $(INSTALL) -c $$f $(DESTDIR)$(XFIGLIBDIR)/bitmaps ; \
+	      $(INSTALL) -c $$f $(DESTDIR)$(XFIGLIBDIR) ; \
 	done) ; 
 
 install.all::
