--- transfig.3.2.5_alpha7/fig2dev/Imakefile.orig	2005-10-27 18:56:02.000000000 +0200
+++ transfig.3.2.5_alpha7/fig2dev/Imakefile	2006-10-28 00:17:31.890948500 +0200
@@ -34,7 +34,7 @@
 XCOMM You should point XFIGLIBDIR to the same directory you did when you compiled
 XCOMM and installed xfig.
 
-XFIGLIBDIR =	/usr/local/lib/X11/xfig
+XFIGLIBDIR =	/usr/share/xfig
 
 XCOMM ******
 XCOMM If your system doesn't have the strerror() function (has sys_errlist) then
@@ -62,7 +62,7 @@
 XCOMM the full name (rgb.txt) now.
 XCOMM
 
-RGB = $(LIBDIR)/rgb.txt
+RGB = /usr/share/X11/rgb.txt
 
 XCOMM Comment out the following #define USEPNG if you don't want PNG support
 XCOMM Uncomment the #define for USEPNG if you want to be able to import
@@ -76,7 +76,7 @@
 #ifdef USEPNG
 PNGLIBDIR = $(USRLIBDIR)
 ZLIBDIR = $(USRLIBDIR)
-PNGINC = -I/usr/include/X11
+PNGINC =
 #endif
 
 XCOMM ****************
@@ -141,7 +141,7 @@
 
 #ifdef I18N
 I18N_DEFS = -DI18N 
-FIG2DEV_LIBDIR = /usr/local/lib/fig2dev
+FIG2DEV_LIBDIR = /usr/share/fig2dev
 I18N_DEV_DEFS = $(I18N_DEFS) -DFIG2DEV_LIBDIR=$(FIG2DEV_LIBDIR) -DFIG2DEV_LIBDIR_STR=\\\"$(FIG2DEV_LIBDIR)\\\"
 
 #endif /* I18N */
@@ -150,7 +150,7 @@
 XCOMM *** You shouldn't have to change anything below this point ***
 XCOMM **************************************************************
 
-DIR_DEFS=	-DBITMAPDIR=\\\"$(DESTDIR)$(XFIGLIBDIR)/bitmaps\\\"
+DIR_DEFS=	-DBITMAPDIR=\\\"$(DESTDIR)$(XFIGLIBDIR)\\\"
 
 #ifdef USEPNG
 DUSEPNG = -DUSE_PNG
@@ -225,11 +225,11 @@
 XCOMM Install the tk bitmaps here
 install::
 	@(cd bitmaps ; \
-	if [ -d $(DESTDIR)$(XFIGLIBDIR)/bitmaps ]; then set +x; \
-	else (set -x; $(MKDIRHIER) $(DESTDIR)$(XFIGLIBDIR)/bitmaps ); fi ; \
-	echo Copying bitmap files for tk to $(DESTDIR)$(XFIGLIBDIR)/bitmaps ; \
+	if [ -d $(DESTDIR)$(XFIGLIBDIR) ]; then set +x; \
+	else (set -x; $(MKDIRHIER) $(DESTDIR)$(XFIGLIBDIR) ); fi ; \
+	echo Copying bitmap files for tk to $(DESTDIR)$(XFIGLIBDIR) ; \
 	for f in * ; do \
-	      $(INSTALL) -c $$f $(DESTDIR)$(XFIGLIBDIR)/bitmaps ; \
+	      $(INSTALL) -c $$f $(DESTDIR)$(XFIGLIBDIR) ; \
 	done) ; 
 
 install.all::
